CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# DEV
+
+## Changes
```

Comment:
It is not "change" as it backward compatible so it is a new feature.

Reply:
Fixed.